rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -33,6 +33,10 @@ load_module /etc/nginx/modules/ngx_http_auth_digest_module.so;
 load_module /etc/nginx/modules/ngx_http_modsecurity_module.so;
 {{ end }}
 
+{{ if (shouldLoadOpenTelemetryModule $cfg $servers) }}
+load_module /etc/nginx/modules/opentelemetry_nginx.so;
```

Comment:
```suggestion
load_module /etc/nginx/modules/otel_ngx_module.so;
```
From experience when building from source the output seems to be named `otel_ngx_module.so`. Should the default name be used instead?